pelican/tests/test_importer.py

```diff
@@ -84,10 +85,12 @@ def test_dircat(self):
         with temporary_folder() as temp:
             fnames = list(silent_f2p(test_posts, 'markdown',
                                      temp, dircat=True))
+        settings = read_settings()
+        subs = settings['SLUG_REGEX_SUBSTITUTIONS']
```

Comment:
You don't need to do `read_settings()` if you don't use the full settings. This could easily be:
```python
subs = DEFAULT_CONFIG['SLUG_REGEX_SUBSTITUTIONS']
```

[Same goes for all the other similar cases below]

Reply:
True, will change.